ceph.spec.in

```diff
@@ -103,6 +98,15 @@ BuildRequires:	sharutils
 %endif
 
 %if 0%{defined suse_version}
+BuildRequires:	python-sphinx
+BuildRequires:	net-tools
+BuildRequires:	xmlstarlet
+BuildRequires:	yasm
+BuildRequires:	libbz2-devel
+BuildRequires:	python-sphinx
+BuildRequires:	net-tools
+BuildRequires:	sharutils
+BuildRequires:	git
```

Comment:
With this change, there's a lot of duplication between what SUSE BuildRequires and what RHEL BuildRequires, so `xmlstarlet`, `yasm`, etc are listed twice in `ceph.spec.in` now. We could probably do another PR to merge these two lists.